doc/source/whatsnew/v0.21.0.txt

```diff
@@ -130,7 +130,7 @@ Other Enhancements
 - `read_*` methods can now infer compression from non-string paths, such as ``pathlib.Path`` objects (:issue:`17206`).
 - :func:`pd.read_sas()` now recognizes much more of the most frequently used date (datetime) formats in SAS7BDAT files (:issue:`15871`).
 - :func:`DataFrame.items` and :func:`Series.items` is now present in both Python 2 and 3 and is lazy in all cases (:issue:`13918`, :issue:`17213`)
-
+- :func: groupby.is_monotonic_increasing and .is_monotonic_decreasing extend Series.is_monotonic_increasing to groups, returning whether each group is monotonically increasing or decreasing, respectively. (:issue:`17015`)
```

Comment:
this won't work for ``:func:`` as you have it written, I think enough is to say ``.is_monotonic_increasing/decreasing`` is added to ``.groupby()``

Reply:
Thanks for this fix and the above test location, which now seems painfully obvious. Will patch up the PR and fix tomorrow.